.github/workflows/publish-release.yml

```
@@ -108,17 +108,17 @@ jobs:
-          skip-dirs: 'ballerina/runtime/examples'
-          format: 'table'
-          exit-code: '1'
+          image-ref: "ballerina/ballerina:release-test"
+          skip-dirs: "ballerina/runtime/examples"
+          format: "table"
+          exit-code: "1"
           timeout: "10m0s"
       - name: cosign-installer
         uses: sigstore/cosign-installer@v3.5.0
       - name: Set up Node.js
         uses: actions/setup-node@v2
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)